universe/Special.cpp

```diff
@@ -58,6 +58,28 @@ Special::~Special() {
     delete m_location;
 }
 
+std::string Special::Description() const
+{
+    std::stringstream result;
+
+    result << UserString(m_description) << "\n";
+
+    for (std::vector<boost::shared_ptr<Effect::EffectsGroup> >::const_iterator it = m_effects.begin();
+         m_effects.end() != it; ++it)
+    {
+        const std::string& description = (*it)->GetDescription();
+
+        if (!description.empty())
+        {
+            result << "\n"
+                   << UserString(description) << "\n";
+        }
+    }
+
```

Comment:
Line 78 and line 79 are empty. It is not common to add two empty lines before return.